### PR TITLE
create on-boarding page and style it.

### PR DESCRIPTION
**Pull request by ahlam377**
relates #8

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,5 +7,23 @@
     <link rel="stylesheet" href="assets/css/style.css" />
     <title>Onboarding</title>
   </head>
-  <body></body>
+  <body>
+    <h1 class="heade">Finance</h1>
```
Comment (moustf) [inline index.html:11]: Financy

File: index.html [hunk 2]
```diff
@@ -7,5 +7,23 @@
...
+    <img src="img.jpg" alt="">
```
Comment (moustf) [inline index.html:12]: fill the alt attr with text to do it in the best practices.

File: index.html [hunk 3]
```diff
@@ -7,5 +7,23 @@
...
+    
+    <div>
+      <h2>save money</h2>
+      <p1>Have all your finances </p1><br/>
```
Comment (moustf) [inline index.html:16]: the p element is a Block-level element, so it will add a line break by default.
no need for <br/>

**Review by moustf — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (muhannaAtalla20) [reply index.html:16]: <p></p> not <p1></p1>

File: index.html [hunk 4]
```diff
@@ -7,5 +7,23 @@
...
+      <p2>in one place!</p2>
```
Comment (muhannaAtalla20) [inline index.html:17]: paragraph tag not content number 2

**Review by muhannaAtalla20 — APPROVED** [review]

**Review by moustf — DISMISSED** [review]
Couldn't be better👏

**Review by MohammedAbuSamra — CHANGES_REQUESTED** [review]
Request changes

File: assets/css/style.css [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+text-align: center;
+font-weight: bold;
+color: white;
+
+
+
+
+
+}
+
```
Comment (moustf) [inline assets/css/style.css:86]: There are lots of empty lines, format the files by Prettier, click alt+shift+f

File: index.html [hunk 6]
```diff
@@ -7,5 +7,23 @@
     <link rel="stylesheet" href="assets/css/style.css" />
     <title>Onboarding</title>
   </head>
-  <body></body>
+
+  <body>
+    <div>
```
Comment (moustf) [inline index.html:12]: You need to put class-"container" for the div.

File: index.html [hunk 7]
```diff
@@ -7,5 +7,23 @@
...
+      
+      <h1><strong>Financy</strong></h1>
```
Comment (moustf) [inline index.html:14]: It's always better to control the styles through CSS code, you can put a font-weight: bolder; to the h1 tag.
No need for <strong> element.

File: index.html [hunk 8]
```diff
@@ -7,5 +7,23 @@
...
+      <p>Have all your finances </p>
+      <p>in one place!</p>
+      
+
+      <footer>
+        <button type="button" class="continue">continue</button>
+      </footer>
+
+    </div>
+  </body>
```
Comment (moustf) [inline index.html:28]: Also, try to format it with Prettier extension.

**Review by moustf — CHANGES_REQUESTED** [review]

File: index.html [hunk 9]
```diff
@@ -1,11 +1,27 @@
...
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="assets/css/style.css" />
-    <title>Onboarding</title>
-  </head>
-  <body></body>
-</html>
+
```
Comment (moustf) [inline index.html:3]: Put these two tags here
<!DOCTYPE html>
<html>

File: index.html [hunk 10]
```diff
@@ -1,11 +1,27 @@
...
+  <link rel="stylesheet" href="assets/css/style.css" />
+  <title>Onboarding</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com/%22%3E">
+  <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;700;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <h4>Financy</h4>
+    <img src="img.jpg" alt="Onboarding design picture">
```
Comment (moustf) [inline index.html:17]: The image should be in the assets/images, not in the root.

File: index.html [hunk 11]
```diff
@@ -1,11 +1,27 @@
...
+    <h2>save money</h2>
+    <div class="text">
+      <p> Have all your finances<br> in one place!</p></div>
```
Comment (moustf) [inline index.html:20]: The <br> tag isn't a good practice, you can use two <p> elements.

File: assets/css/style.css [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+.container{
+    text-align: center;
+    position:absolute;
```
Comment (moustf) [inline assets/css/style.css:4]: I think it will be better if you center the container vertically on the page.

File: index.html [hunk 13]
```diff
@@ -1,11 +1,27 @@
...
+   <footer>
+      <button type="button" class="continue">continue</button>
```
Comment (moustf) [inline index.html:22]: The continue button should take to the home page, wrap the button inside a <a> tag to do that

**Review by moustf — CHANGES_REQUESTED** [review]

**Review by MohammedAbuSamra — APPROVED** [review]
Awesome, thank you.

**Review by muhannaAtalla20 — APPROVED** [review]

**Review by moustf — APPROVED** [review]

**Pull request merged**